bin/run-tests.sh

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env sh
```

Comment:
Would it be cool to rename `bin/run-tests-in-docker.sh` and `bin/run-tests.sh` => `bin/run-smoke-tests-in-docker.sh` and `bin/run-smoke-tests.sh`? 
The Elixir tests and the smoke test happen to pretty much do the same thing, but I would prefer keeping the distinction.

Reply:
Well, we could, but I would personally like to keep its current name for consistency with other repos.

Reply:
Right, I realized later that this was the norm. Ok that's fine :)